core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -288,6 +263,9 @@ class RequestChannel(val numProcessors: Int, val queueSize: Int) extends KafkaMe
   def shutdown() {
     requestQueue.clear()
   }
+
+  def sendShutdownRequest(): Unit = requestQueue.put(ShutdownRequest)
```

Comment:
I used that name initially, but since it is invoked from `KafkaRequestHandler`, which is also responsible for receiving the shutdown request, it seemed clearer with this name.